corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -570,6 +571,35 @@ FormplayerFrontend.on('refreshApplication', function (appId) {
     });
 });
 
+/**
+ * breakLocks
+ *
+ * Sends a request to formplayer to wipe out all application and user db for the
+ * current user. Returns the ajax promise.
+ */
+FormplayerFrontend.reqres.setHandler('breakLocks', function () {
+    var user = FormplayerFrontend.request('currentUser'),
+        formplayer_url = user.formplayer_url,
+        resp,
+        options = {
+            url: formplayer_url + "/break_locks",
+            data: JSON.stringify({
+                domain: user.domain,
+                username: user.username,
+                restoreAs: user.restoreAs,
+            }),
+        };
+    Util.setCrossDomainAjaxOptions(options);
+    formplayerLoading();
+    resp = $.ajax(options);
+    resp.fail(function () {
+        formplayerLoadingComplete(true);
+    }).done(function (response) {
+        breakLocksComplete(response.hasOwnProperty('exception'), response.message);
```

Comment:
Why don't you need to call `formplayerLoadingComplete()` here?

Reply:
Hm, I was just following the pattern from clearUserDataComplete down below.

breakLocksComplete() calls hideLoading() and displays a message. I think the only difference is that formplayerLoadingComplete has a different default message, although I could possibly consolidate to make the code less verbose

Reply:
Cool, makes sense. I had looked around at where else `formplayerLoadingComplete ` is called and now realize I was looking as `refreshApplication`. Agree that making this match `clearUserDataComplete` sounds right.